### ISSUE TITLE
Duda sobre normalizar en parte 1

### ISSUE THREAD

**Issue by valentin-diaz** [Exploratorio-DCC-PUC/Syllabus#596]
Hola, en la parte 1 se pide "normalizar las características", pero en la documentación de sk-learn hay varias formas distintas de re-escalar los datos y, por lo que he leído, se recomiendan formas distintas de escalar dependiendo del modelo de Machine Learning que se usa. ¿Necesariamente hay que usar el método .normalizer() para ambos o se busca que nos decidamos por alguno en particular?

También en la misma pregunta, se dice que se espera "UNA separación donde el vector de clase corresponda al rango de
precios en el caso del clasificador y memoria interna en el caso del regresor", pero no entiendo a qué se refiere con el énfasis en "UNA separación". ¿No se supone que hay que hacer una separación distinta para elegir el vector del clasificador y otra para el del regresor?

**Comment by felipegarcia98** [comment]
Hola 😄 /
efectivamente en la [documentación de preprocesamiento de SKLearn](https://scikit-learn.org/stable/modules/preprocessing.html) hay múltiples alternativas de preprocesamiento, entre las que se encuentran diversos escalamientos y la normalización. La idea realmente es que uds investiguen y seleccionen lo que encuentren mejor, por lo que pueden no usar normalize si consideran (y justifican) otros métodos (igualmente hay que justificar el tipo de normalización, así que no hay gran cambio). En estricto rigor, lo más correcto sería lo que tu propones, pero para simplificarles la vida les acotamos las opciones, pero encantado de que ocupes otro método si tu análisis de los datos señala que sería más correcto realizar otro pre procesamiento

[Pequeña explicación de las diferencias entre escalamiento y normalización](https://www.kaggle.com/rtatman/data-cleaning-challenge-scale-and-normalize-data)
[Más profunda explicación](https://www.analyticsvidhya.com/blog/2020/04/feature-scaling-machine-learning-normalization-standardization/)

Sobre la segunda pregunta, realmente era para recalcar que no hicieran más cosas de lo solicitado. Es decir, que separen los datos para que haya un `Y=df['price_range']` para el clasificador y un `Y=df['int_memory']` para el regresor. Lamentamos que no se haya entendido uwu
